beam.py

```diff
@@ -1,3 +1,5 @@
+""" This file is all the Beam related stuff """
```

Comment:
Doesn't really explain what the file does. Maybe change to: `Connects to Beam's chat and liveloading.`?